data/constants/keycodes/keycodes_0.0.1_quantum.hjson

```diff
@@ -507,6 +507,20 @@
             "aliases": [
                 "AC_TOGG"
             ]
+        },
```

Comment:
You should not add to this file, its effectively locked in now its been merged to master. A `data/constants/keycodes/keycodes_0.0.2_quantum.hjson` should be created, its contents will be merged at build time.

Reply:
thanks